tasks/selinux_load_module.yml

```diff
@@ -0,0 +1,45 @@
+- name: Set defaults
+  set_fact:
+    name: "{{ item.name | default('')}}"
+    state: "{{ item.state | default('enabled') }}"
+    priority: "{{ item.priority | default('400') }}"
+    store: "{{ selinux_policy | default(ansible_selinux.type, true) }}"
```

Comment:
These variables are global and will persist after the call to the role.  Instead, add `vars:` to the `include_tasks` above.